app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+    belongs_to :post, foreign_key: 'post_id'
+    belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+
+    def update_like_counter
```

Comment:
- [ ] The same approach, I suggest the same to other update methods by putting them after `private`

Reply:
- [ ] As mentioned by the former reviewer, kindly make the `update_like_counter` a private method. This will ensure that outside methods can't update the `likes_counter` but done automatically through this method. 👍 💻